tests/functional/Config.php

```diff
@@ -829,4 +829,18 @@ public function testAutoCreateInfocom()
             $this->boolean($infocom_exists)->isFalse();
         }
     }
+
+    public function testEmptyUpdate()
+    {
+        \Config::setConfigurationValues('core', ['maintenance_text' => 'test']);
+        // If not handled, this second call would trigger an exception if handled at the DBAL level as the params array would be empty.
+        // If not handled by GLPI at all, it will trigger a SQL syntax error.
+        // Need to specify a field to get the execution to the point the error could occur.
+        // The null value will cause the entry to be removed and an empty array to be passed to the DBAL.
+        \Config::setConfigurationValues('core', ['maintenance_text' => null]);
```

Comment:
This behaviour does not seems normal to me. It should result in an `SET maintenance_text = NULL` query.

Reply:
See #15720.